package.json

```diff
@@ -58,7 +55,12 @@
     "karma-jasmine": "^1.0.2",
     "karma-jasmine-html-reporter": "^0.2.2",
     "protractor": "^3.3.0",
-    "rimraf": "^2.5.4"
+    "rimraf": "^2.5.4",
+
+    "@types/core-js": "^0.9.34",
+    "@types/jasmine": "^2.5.35",
+    "@types/node": "^6.0.45",
+    "@types/protractor": "^1.5.20"
```

Comment:
Compilation didn't work until I added this: "@types/selenium-webdriver": "2.44.26"
Isn't it missing?